source/jormungandr/jormungandr/parking_space_availability/car/common_car_park_provider.py

```diff
@@ -44,6 +44,7 @@
 class CommonCarParkProvider(AbstractParkingPlacesProvider):
     def __init__(self, url, operators, dataset, timeout, feed_publisher, **kwargs):
 
+        AbstractParkingPlacesProvider.__init__(self, **kwargs)
```

Comment:
For python2 (:disappointed:) it's more:
```suggestion
        super(CommonCarParkProvider, self).__init__(self, **kwargs)
```